.github/workflows/python_test.yml

```diff
@@ -0,0 +1,40 @@
+name: Python application test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install -e .
+        pip3 install absl-py pytest Pillow
+        sudo curl -sSL http://get.gazebosim.org | sh
+        sudo apt install python3-tk xvfb
+        git submodule update --init --recursive
+        mkdir build
+        cd build
+        cmake ..
+        make -j
+        cd ..
+        pip3 install -e .
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # distable static check for now
+        # stop the build if there are Python syntax errors or undefined names
+        # flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        # flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        GAZEBO_MODEL_PATH=./python/social_bot/models xvfb-run python python/social_bot/envs/play_ground_test.py
```

Comment:
Is it possible to run all testes by `GAZEBO_MODEL_PATH=./python/social_bot/models xvfb-run python -m unittest discover -p "*_test.py" -v` ?